graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/inputs/Extractor.java

```diff
@@ -124,7 +124,7 @@ public static Type fuzzyValueOf(String s) {
     public Extractor(MetricRegistry metricRegistry,
                      String id,
                      String title,
-                     int order,
+                     long order,
```

Comment:
This was done because inconsistent data types were used all over the place, so the largest one was chosen.